Makefile

```diff
@@ -18,12 +19,12 @@ dependencies:
 	luarocks make --local rockspec
 
 build:
-	$(DOCKER_COMPOSE) build
+	$(S2I) build . openresty-builder docker-gateway-test
```

Comment:
I'd much rather depend on some published version. At least for now it would make things much easier.